internal/publish/publish.go

```diff
@@ -183,6 +184,8 @@ func (h *PublishHandler) process(ctx context.Context, data *verifyapi.Publish, b
 	metrics := h.serverenv.MetricsExporter(ctx)
 	metricsMiddleWare := metricsware.NewMiddleWare(&metrics)
 
+	logger.Info("publish API request", zap.String("HealthAuthorityID", data.HealthAuthorityID))
```

Comment:
```suggestion
	logger.Infow("publish API request", "health_authority_id", data.HealthAuthorityID)
```

Note the `w` and snake case (works better for our logger)

Reply:
Ah actually, after seeing the rest of the code, I think we should jump up to L183 and change it to:

```go
logger := logging.FromContext(ctx).With("health_authority_id", data.HealthAuthorityID)
```

and then leave the rest of this code as-was.

Reply:
Whoa, I had no idea our logger was so awesome. You got it!

Reply:
Yea, `.Named` and `.With` create new loggers and propagate the context through. So it'll add that field to all logger.Something calls